### PR TITLE
added solutions

### PR DESCRIPTION
**Pull request by engineer027**

### CODE REVIEW

File: src/main/java/mate/academy/controllers/AddDriverToCarController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.lib.Injector;
+import mate.academy.model.Car;
+import mate.academy.model.Driver;
+import mate.academy.service.CarService;
+import mate.academy.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    CarService carService = (CarService) injector.getInstance(CarService.class);
```
Comment (RuslanIbIsh) [inline src/main/java/mate/academy/controllers/AddDriverToCarController.java]: сделай его private final - было в чеклисте

File: src/main/webapp/WEB-INF/views/drivers/drivers.jsp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            </td>
+            <td>
+                <c:out value="${driver.licenceNumber}"/>
+            </td>
+            <td>
+                <a href="${pageContext.request.contextPath}/drivers/delete?id=${driver.id}">Delete</a>
+            </td>
+        </tr>
+    </c:forEach>
+
```
Comment (RuslanIbIsh) [inline src/main/webapp/WEB-INF/views/drivers/drivers.jsp]: лишняя строка)))

**Review by RuslanIbIsh — CHANGES_REQUESTED** [review]

**Review by RuslanIbIsh — APPROVED** [review]

File: src/main/java/mate/academy/controllers/CreateCarController.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String model = req.getParameter("model");
+        Long manufacturerId = Long.valueOf(req.getParameter("manufacturerId"));
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/controllers/CreateCarController.java]:
```suggestion
        Long manufacturerId = Long.valueOf(req.getParameter("manufacturer_id"));
```
let's use the same approach everywhere for request parameters

File: src/main/java/mate/academy/controllers/CreateDriverController.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String licenceNumber = req.getParameter("licenceNumber");
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/controllers/CreateDriverController.java]:
```suggestion
        String licenceNumber = req.getParameter("licence_number");
```

File: src/main/webapp/WEB-INF/views/cars/create.jsp [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+</head>
+<body>
+<h1>Please provide your car details</h1>
+<h4 style="color: green">${message}</h4>
+<form method="post" action="${pageContext.request.contextPath}/cars/create">
+    Please provide model name<br>
+    <input type="text" name="model" required><br>
+    Please provide manufacturer id<br>
+    <input type="number" name="manufacturerId" required><br>
+    <input type="reset">
```
Comment (KseniiaMakarova) [inline src/main/webapp/WEB-INF/views/cars/create.jsp:14]: `<input type="reset">` what is this for? and how does it look?

File: src/main/webapp/WEB-INF/views/cars/drivers/add.jsp [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create car</title>
```
Comment (KseniiaMakarova) [inline src/main/webapp/WEB-INF/views/cars/drivers/add.jsp]:
```suggestion
    <title>Add driver to car</title>
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]
there only a few comments above

**Pull request merged**